### DISCUSSION TITLE
EPC server prevents emacs exit without prompt

### DISCUSSION THREAD

**Discussion by toweringwill** [emacs-eaf/emacs-application-framework#908] [Q&A]
After install and config of eaf, new network services are running on 127.0.0.1 in emacs, preventing me from exiting the app with C-x C-c without prompt. 

1. I do not understand why these are running, what is their purpose?
2. I do not know enough elisp to override shutdown behavior for just these services. (I do not want to inhibit the prompt for any service that might be running, just for these that run every time with eaf)

**Comment by manateelazycat** [comment]
(setq confirm-kill-processes nil)

**Comment by MatthewZMD** [comment]
> 1. I do not understand why these are running, what is their purpose?

It is the backend of EAF.
